distribution/sdk/sample-mpegts-streamgenerator/src/main/java/org/codice/alliance/distribution/sdk/video/stream/mpegts/MpegTsUdpClient.java

```diff
@@ -55,7 +55,12 @@
  */
 public class MpegTsUdpClient {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(MpegTsUdpClient.class);
+    private static final Logger LOGGER;
+
+    static {
```

Comment:
This doesn't give an integrator/deployer much opportunity to quiet this logging if they decide they don't care (short of at runtime). Shouldn't we do this through logging configuration as we do with everything else where we want to set initial levels? For example, [here is where DDF sets custom log levels](https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg#L93).

You'd need to override that config file in Alliance to get that, but it would allow admins to change logging as needed on deployments.